src/main/java/build/buildfarm/worker/UploadManifest.java

```diff
@@ -163,29 +163,16 @@ public void addContent(ByteString content, CASInsertionPolicy policy, Consumer<B
   }
 
   private void addFile(Path file, long size, CASInsertionPolicy policy) throws IOException {
```

Comment:
Do you still need to pass the size externally? I think DigestUtil should handle computing the digest correctly, regardless of whether the file is a symlink or not.

Reply:
nope, cleaning that up.